src/Compilers/CSharp/Test/Emit/CodeGen/SwitchTests.cs

```diff
@@ -7125,7 +7125,7 @@ .maxstack  2
   IL_0760:  ret
 }";
             compVerifier.VerifyIL("ConsoleApplication24.Program.IsWarning", codeForSwitchStatement);
-            compVerifier.VerifyIL("ConsoleApplication24.Program.IsWarning_IsExpression", codeForExpression);
+            compVerifier.VerifyIL("ConsoleApplication24.Program.IsWarning_IsExpression", codeForSwitchStatement);
```

Comment:
>codeForSwitchStatement [](http://example.com/codeflow?start=89&length=22)

Could you explain why this change is expected?

Reply:
While `return` is spelled out in the switch statement, we have a unconditional jump to `ret` for is-expression which will get inlined. When this is done for switch, all these are expected to produce the same output.